src/cli/domain/get-components-deps.js

```diff
@@ -43,7 +43,7 @@ module.exports = function(components){
   });
 
   return {
-    modules: _.keys(deps.modules),
+    modules: _.union(coreModules, _.keys(deps.modules)).sort(),
```

Comment:
This is used for the CLI. Now the cli by default allows any core module to be used. When running in the registry, still, the dependency will need to be whitelisted as all the other deps in order to be usable.